.github/workflows/cypress.yml

```diff
@@ -77,9 +77,10 @@ jobs:
         with:
           install: false
           browser: chrome
+          headed: true
```

Comment:
Unfortunately, running cypress in headless mode causes the bug where the single PDF won't be able to get the content from an external API, so I had to run this in headed mode. I think this is fine as far as regression, but please let me know if there are reasons we should run them in headless!

Reply:
I think headed is fine!